src/sidebar.cpp

```diff
@@ -315,9 +315,9 @@ void player::disp_status( const catacurses::window &w, const catacurses::window
     }
 
     wmove( w, sideStyle ? 1 : 2, 0 );
-    if( get_hunger() > 2800 ) {
+    if( get_hunger() >= 400 and get_starvation() > 2400 ) {
         wprintz( w, c_red,    _( "Starving!" ) );
-    } else if( get_hunger() > 1400 ) {
+    } else if( get_hunger() >= 400 and get_starvation() > 1000 ) {
```

Comment:
Use `&&` instead of `and` here as logical AND.

Reply:
Will do.

Reply:
This should be one of the reasons why VS build is failing in AppVeyor - VC++ doesn't respect standards.